### PR TITLE
6252 account type counts

### PR DESCRIPTION
**Pull request by gdicristofaro**

### CODE REVIEW

**Comment by jonas-koeritz** [comment]
Whats the 4-digit number in the PR title referring to?

**Comment by rcordovano** [comment]
The four digit number is a cross reference to a "story" in our internal Jira instance at Basis Technology.

**Comment by jonas-koeritz** [comment]
Thanks, was just wondering if i missed an issue tracker or something where I should have listed my new PR.

File: Core/src/org/sleuthkit/autopsy/datamodel/accounts/Accounts.java [hunk 1]
```diff
@@ -249,12 +255,69 @@ public String getItemType() {
...
+            return counts.get(accountType);
+        }
+        
+        /**
+         * Retrieves an alphabetically organized list of all the account types.
+         * @return      An alphabetically organized list of all the account types.
+         */
+        List<String> getTypes() {
+            List<String> types = new ArrayList(counts.keySet());
+            Collections.sort(types);
```
Comment (wschaeferB) [inline Core/src/org/sleuthkit/autopsy/datamodel/accounts/Accounts.java:284]: This method causes several build warning.

Comment (gdicristofaro) [reply Core/src/org/sleuthkit/autopsy/datamodel/accounts/Accounts.java:284]: Ok.  I believed this is fixed.

**Pull request merged**